dapr/clients/grpc/client.py

```diff
@@ -1088,6 +1088,226 @@ def unlock(
         return UnlockResponse(status=UnlockResponseStatus(response.status),
                               headers=call.initial_metadata())
 
+    def start_workflow(
+            self,
+            instance_id: str,
+            workflow_component: str,
+            workflow_name: str,
+            input: bytes,
+            workflow_options: dict) -> api_v1.StartWorkflowResponse:
+        """Starts a workflow.
+
+            Args:
+                instance_id (str): the name of the workflow instance,
+                                    e.g. `order_processing_workflow-103784`.
+                workflow_component (str): the name of the workflow component
+                                    that will run the workflow. e.g. `dapr`.
+                workflow_name (str): the name of the workflow that will be executed.
+                input (bytes): the input that the workflow will receive.
+                workflow_options (dict): the key-value options that the workflow will receive.
+
+            Returns:
+                :class:`StartWorkflowResponse`: Instance ID associated with the started workflow
+        """
+        # Warnings and input validation
+        warn('The Workflow API is an Alpha version and is subject to change.',
+             UserWarning, stacklevel=2)
+        validateNotBlankString(instance_id=instance_id,
+                               workflow_component=workflow_component,
+                               workflow_name=workflow_name)
+        # Actual start workflow invocation
+        req = api_v1.StartWorkflowRequest(
+            instance_id=instance_id,
+            workflow_component=workflow_component,
+            workflow_name=workflow_name,
+            options=workflow_options,
+            input=input)
+        response, call = self._stub.StartWorkflowAlpha1.with_call(req)
+
+        return api_v1.StartWorkflowResponse(instance_id=response.instanceID)
```

Comment:
This should return the StartWorkflowResponse from your response helpers - but you also need to import that.